jdbc/src/main/java/io/kestra/jdbc/repository/AbstractJdbcCrudRepository.java

```diff
@@ -97,13 +95,18 @@ public int saveBatch(List<T> items) {
      * It uses an update statement, so the item must be already present in the database.
      */
     public T update(T current) {
+        String uid = current instanceof HasUID hasUID ? hasUID.uid() : null;
+
+        if(uid == null)
+            throw new IllegalArgumentException("Cannot update entity: unknown type '" + current.getClass().getName() + "'");
+
```

Comment:
I would prefer a more straightforward implementation lilke this one:

```suggestion
        if (! current instantof HasUID hasUID) {
            throw new IllegalArgumentException("Cannot update entity: ' + current.getClass().getName() + "'  didn't implements HasUID");
        }
        String uid = hasUID.uid();
```

Reply:
Acknowledged the changes. Thanks for suggestion